rahat/helloworld-DEV-43/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="./app.js"></script>
```

Comment:
I need you to add defer attribute to script.

Reply:
resolved.

Reply:
LGTM